File: win-prog/ProgWinCS/Hello Windows Forms/InheritWithConstructor/InheritWithConstructor.cs
//-----------------------------------------------------
// InheritWithConstructor.cs © 2001 by Charles Petzold
//-----------------------------------------------------
using System;
using System.Drawing;
using System.Windows.Forms;

class InheritWithConstructor: Form
{
     public static void Main()
     {
          Application.Run(new InheritWithConstructor());
     }
     public InheritWithConstructor()
     {
          Text = "Inherit with Constructor";
          BackColor = Color.White;
     }
}

File: win-prog/ProgWinVB/Hello Windows Forms/InheritWithConstructor/InheritWithConstructor.vb
'-------------------------------------------------------
' InheritWithConstructor.vb (c) 2002 by Charles Petzold
'-------------------------------------------------------
Imports System
Imports System.Drawing
Imports System.Windows.Forms

Module InheritWithConstructor
    Sub Main()
        Application.Run(New InheritAndConstruct())
    End Sub
End Module

Class InheritAndConstruct
    Inherits Form

    Sub New()
        Text = "Inherit with Constructor"
        BackColor = Color.White
    End Sub
End Class
